src/main/java/uk/gov/hmcts/ccd/data/casedetails/CaseAuditEventMapper.java

```diff
@@ -66,6 +79,19 @@ public CaseAuditEventEntity modelToEntity(final AuditEvent auditEvent) {
         newCaseAuditEventEntity.setEventName(auditEvent.getEventName());
         newCaseAuditEventEntity.setSummary(auditEvent.getSummary());
         newCaseAuditEventEntity.setDescription(auditEvent.getDescription());
+        if(auditEvent.getSignificantItem()!=null) {
```

Comment:
Formatting:
* Missing space after `if`
* Missing spaces on either side of `!=`

Reply:
Addressed